packages/hub/routes/data-integrity-checks.ts

```diff
@@ -0,0 +1,59 @@
+import Koa from 'koa';
+import autoBind from 'auto-bind';
+import { inject } from '@cardstack/di';
+import DataIntegrityChecksScheduledPayments, {
+  IntegrityCheckResult,
+} from '../services/data-integrity-checks/scheduled-payments';
+import config from 'config';
+
+export default class DataIntegrityChecksRoute {
+  dataIntegrityCheckScheduledPayments: DataIntegrityChecksScheduledPayments = inject(
+    'data-integrity-checks-scheduled-payments',
+    {
+      as: 'dataIntegrityCheckScheduledPayments',
+    }
+  );
+
+  constructor() {
+    autoBind(this);
+  }
+
+  async get(ctx: Koa.Context) {
+    if (ctx.query.secret !== config.get('checkly.webhookSecret')) {
```

Comment:
Can we put the secret somewhere besides the URL, like a header for example? I'm concerned about the secret getting logged.

Reply:
Nice catch and I agree. I moved the authorization to a header.